docs/gettingstarted/install-landscaper-controller.md

```diff
@@ -16,9 +18,8 @@ helm install -n ls-system landscaper ./charts/landscaper
 We are also building OCI charts so a specific version of Landscaper can be installed with:
 
 ```
-export HELM_EXPERIMENTAL_OCI=1
 export LS_VERSION="v0.13.0" # use the latest available version
-helm chart pull eu.gcr.io/gardener-project/landscaper/charts/landscaper-controller:$LS_VERSION
+helm pull eu.gcr.io/gardener-project/landscaper/charts/landscaper-controller:$LS_VERSION
 helm chart export eu.gcr.io/gardener-project/landscaper/charts/landscaper-controller:$LS_VERSION ./charts
 helm install ./charts
```

Comment:
```suggestion
helm install landscaper-controller landscaper-controller-${LS_VERSION}.tgz
```

Reply:
Don't we need a path to the tgz file here?

Reply:
The first parameter is the release name. The secnd one is the path to the tgz file.